Package.swift

```diff
@@ -2,50 +2,356 @@
 // The swift-tools-version declares the minimum version of Swift required to build this package.
 
 import PackageDescription
+import Foundation
+
+let streamChatFiles: [String] = [
```

Comment:
Are they magically generated or we would need to do it by hands?

Reply:
@DominikBucher12 We need some kind of logic here
pseudocode
```swift
let streamChatSourceFiles = FileManager.files(under: "/Sources/StreamChat").filter { $0.name.suffix == ".swift" && $0.name.suffix != "_Tests.swift" } // or Mock
// same for streamChatUI
```

Reply:
Indeed, this will be generated eventually. The first goal is to make the SPM integration work 😅 Then we can talk about how to improve it.

The snippet you suggest @b-onc is of course our first try. `Package.swift` is not evaluated in the context of the root folder. You can't iterate over the files from the directly. This will have to be generated by some kind of GH action, until SPM has support for `*_Tests.swift` etc.